influxdb/src/test/scala/docs/scaladsl/InfluxDBSourceSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package docs.scaladsl
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.influxdb.scaladsl.InfluxDBSource
+import akka.stream.scaladsl.Sink
+import akka.testkit.TestKit
+import org.influxdb.InfluxDB
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, MustMatchers, WordSpec}
+import org.scalatest.concurrent.ScalaFutures
+import akka.stream.testkit.scaladsl.StreamTestKit.assertAllStagesStopped
+import docs.javadsl.TestUtils._
+import docs.javadsl.TestConstants._
+import org.influxdb.dto.Query
+
+class InfluxDBSourceSpec
+    extends WordSpec
+    with MustMatchers
+    with BeforeAndAfterEach
+    with BeforeAndAfterAll
+    with ScalaFutures {
+
+  val databaseName = "InfluxDBSourceSpec"
```

Comment:
Nit-pick: due to this being a constant, we can follow [the Scala standard](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods) of UpperCamelCasing (and probably using `final`, since this is a class):
```

```final val DatabaseName = ...```